arkane/statmech.py

```diff
@@ -1248,3 +1249,59 @@ def determine_rotor_symmetry(energies, label, pivots):
         logging.info('Determined a symmetry number of {0} for rotor of species {1} between pivots {2}'
                      ' based on the {3}.'.format(symmetry, label, pivots, reason))
     return symmetry
+
+class HinderedRotor2D(Mode):
```

Comment:
It would make most sense to place it in rmgpy.statmech.torsion.pyx, but it creates a cyclic import I could resolve related to loading qm files, I think statmech.py is the best alternative, statmech.mode isn't in the first list of places I'd look for this in.